src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -393,9 +370,7 @@ private boolean stringAlreadyInUseElsewhere(String string, String field) {
     return inUse;
   }
 
-  // function: candidateStringAlreadyInUseElsewhere
-  // purpose: Takes a candidate name or code and checks for conflicts with other name/codes or other
-  //   strings that are already being used in some other way.
+  // checks for conflicts between a candidate name and other name/codes or other reserved strings
   // param: candidateString is a candidate name or code
```

Comment:
did you intend to leave these param comments?

Reply:
yes.  the meaning of 'field' is not obvious